specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/tenantConfiguration.json

```diff
@@ -0,0 +1,256 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01-preview",
+    "title": "portal",
+    "description": "Manage tenant configuration.",
+    "x-ms-code-generation-settings": {
+      "name": "portalClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.Portal/tenantConfigurations": {
```

Comment:
>"/providers/Microsoft.Portal/tenantConfigurations" [](start = 4, length = 50)

Since this is a tenant configuration why do you've a list call?

Reply:
The tenant configuration was recently merged with master in 2019 version. I am just adding it to 2020 version of the RP. As I understand, this is to get all different types of tenant configurations that could be added in the future.